wagtail/admin/templates/wagtailadmin/home/site_summary.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n wagtailadmin_tags %}
 {% if summary_items %}
-    <section class="panel summary nice-padding">
+    <section class="summary">
```

Comment:
we have adopted a `w-summary` here and used the nice padding mixin instead

https://github.com/wagtail/wagtail/pull/8922